thirdeye/thirdeye-frontend/app/pods/manage/alert/controller.js

```diff
@@ -0,0 +1,252 @@
+/**
+ * Controller for Alert Landing and Details Page
+ * @module manage/alert
+ * @exports manage/alert
+ */
+import fetch from 'fetch';
+import Ember from 'ember';
+import moment from 'moment';
+import { checkStatus } from 'thirdeye-frontend/helpers/utils';
+import { task, timeout } from 'ember-concurrency';
+
+export default Ember.Controller.extend({
+  /**
+   * Set up to receive prompt to trigger page mode change.
+   * When replay is received as true, it indicates that this is a
+   * newly created alert and replay is needed in order to display anomaly data.
+   */
+  queryParams: ['replay'],
+  replay: false,
```

Comment:
Please add jsdocs comments for this variable, as per Data UI's coding standards wiki